### ISSUE TITLE
no-property-change-update is incorrect

### ISSUE THREAD

**Issue by LarsDenBakker** [43081j/eslint-plugin-lit#62]
`no-property-change-update` states that changing properties will not trigger a re-render. This is incorrect, as long as you're calling super last it works fine.

This change will be taken along properly:

```js
class MyElement extends LitElement {
  update(changed) {
    this.foo = 'bar';
    super.update(changed);
  }
}
```

This won't:

```js
class MyElement extends LitElement {
  update(changed) {
    super.update(changed);
    this.foo = 'bar';
  }
}
```

Is it possible to detect the call to super? 

This is a prime use case for creating computed properties. Doing this in `updated` will trigger two renders for a single change. As it is I don't think it should be a linting rule, especially not a recommended one, because it will confuse developers.
